services/settings/pkg/store/defaults/defaults.go

```diff
@@ -334,7 +334,7 @@ func generateBundleAdminRole() *settingsmsg.Bundle {
 				Id:          WritePublicLinkPermissionID,
 				Name:        WritePublicLinkPermissionName,
 				DisplayName: "Write publiclink",
-				Description: "This permission permits to write a public link.",
+				Description: "This permission permits to write into a public link.",
```

Comment:
No that is not what this permission is giving. It allows to write (aka create, generate, ....) a public link.

Reply:
```suggestion
				Description: "This permission allows to create public links.",
```